dist/app.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var _express = require('express');
+
+var _express2 = _interopRequireDefault(_express);
+
+var _bodyParser = require('body-parser');
+
+var _bodyParser2 = _interopRequireDefault(_bodyParser);
+
+var _create = require('./routes/create');
+
+var _create2 = _interopRequireDefault(_create);
+
+var _search = require('./routes/search');
+
+var _search2 = _interopRequireDefault(_search);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+// const bodyParser = require('body-parser');
```

Comment:
I might have pushed a little too early. I was working on adding a new feature to the app. I really should have branched out. My bad. I'll fix that.